src/validation/validationRules.js

```diff
@@ -256,7 +256,26 @@ var validations = {
     },
     hasPrice: function hasPrice(values, value, key) {
```

Comment:
For some reason, the validation error for this validator doesn't disappear even though the data in missing languages have been added.

Still, when trying to submit after filling in the data in required languages, submission does succeed, so this is just a display glitch for the validation error.